pkg/storage/fs/owncloud/owncloud.go

```diff
@@ -1589,6 +1589,25 @@ func (fs *ocfs) Move(ctx context.Context, oldRef, newRef *provider.Reference) (e
 	if err = os.Rename(oldIP, newIP); err != nil {
 		return errors.Wrap(err, "ocfs: error moving "+oldIP+" to "+newIP)
 	}
+
+	log := appctx.GetLogger(ctx)
+	conn := fs.pool.Get()
+	defer conn.Close()
+	// Ideally if we encounter an error here we should rollback the Move/Rename.
+	// But since the owncloud storage driver is not being actively used by anyone other
+	// than the acceptance tests we should be fine by ignoring the errors.
+	_ = filepath.Walk(newIP, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			log.Error().Str("path", path).Err(err).Msg("error caching id")
```

Comment:
Can you add a log about rolling back the whole op? And a TODO comment

Reply:
Ok I opened an issue https://github.com/cs3org/reva/issues/1699
And added todos.